### ISSUE TITLE
define the user interactive problem. 

### ISSUE THREAD

**Issue by kailuowang** [A-Noctua/DQN-Sandbox#7]
Is it  non stationary ? Non iid?
Is the q function differentiable? Does it matter?
Can it converge? What if it cannot? Can it still work?
Is it a POMDP?
What's an episode?

Can the action value function be linear so that the least square prediction be solved directly instead of gradient decent (p43 of lecture 6, fa.PDF)? 
Or it is nonlinear so that you have to use neural network with gradient decent. 

Is it better solved with Value based RL or Policy based RL (Lecture 7, PG.pdf p5)

Is it contextual multi arm bandit problem? Would linear UCB be a better solution? or is DQN a better opportunity as it can learn a higher level understanding of the context vs. Linear regression. 

Does it have overfitting problem?
